package.json

```diff
@@ -46,6 +49,7 @@
   },
   "devDependencies": {
     "babel-eslint": "^10.0.2",
+    "babel-plugin-macros": "^2.8.0",
     "eslint": "^6.8.0",
     "eslint-config-react-app": "^5.0.1",
     "eslint-config-react-tools": "^1.1.7",
```

Comment:
Is this a react-scripts related config? Wondering if we should use the same as aragon/aragon (`standard`)?

Reply:
It makes it possible to use Babel macros, which allows to use `styled-components/macro` rather than the styled-component babel plugin, but I reverted back to use babel-plugin-styled-components to have more control over the configuration.